app/assets/javascripts/likes.js

```diff
@@ -0,0 +1,17 @@
+
+
+
+$(document).ready(function() {
+
+  $('.likes-link').on('click', function(event){
+      event.preventDefault();
+
+      var likeCount = $(this).siblings('.likes_count');
+
+      console.log(this)
+
+      $.post(this.href, function(response){
+        likeCount.text(response.new_like_count);
```

Comment:
What you're doing here is setting the text of **all** elements of the class '.likes_count' to the new like count. When really you want to reset only the one that you clicked on. So you need to find a way of setting the text value of only the one that you clicked on.